src/makeCalculator.js

```diff
@@ -1,10 +1,41 @@
 'use strict';
 
 /**
+ * Generates an object with main calculator operations.
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate(operation, value) {
+      this.result = operation(value, this.result);
+
+      return this;
+    },
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(value, result) {
```

Comment:
Let's imagine that this is a calculator and you want to add some number, you press +2, for example, the question is where will you take the result each time to pass it as the second arg?
Methods should take only one argument and take the result that is already stored in memory
In case of add you should increase result on value like `this.result += value`

Reply:
There was no any mentions that these methods (`add, subtract, divide, multiply`) can be executed directly without `operate` method. Example code shows only `operate` case and all the tests passed, so I didn't expect any other cases.